### ISSUE TITLE
[DOCS][Detections] - Update detection rule exceptions export and import

### ISSUE THREAD

**Issue by yctercero** [elastic/security-docs#1170]
## Description 

Starting 7.16, [`exceptions`](https://www.elastic.co/guide/en/security/current/exceptions-api-overview.html) will be exported as part of the rules export. Rules include pointers to what exception lists are associated with them. These pointers were previously always exported, but the export of the rule did not include the exceptions themselves. Now we do!

Example of what new export looks like:

```ndjson
{"id":"d4db8800-30df-11ec-88a5-fb21b48c9b4e","updated_at":"2021-10-19T13:24:05.872Z","updated_by":"elastic","created_at":"2021-10-19T13:24:05.872Z","created_by":"elastic","name":"Rule 1","tags":[],"interval":"30s","enabled":true,"description":"Sample rule with single exception list","risk_score":1,"severity":"high","output_index":".siem-signals-devin-hurley-default","author":[],"false_positives":[],"from":"now-6m","rule_id":"query-with-single-exception-list","max_signals":100,"risk_score_mapping":[],"severity_mapping":[],"threat":[],"to":"now","references":[],"version":1,"exceptions_list":[{"id":"cd62f410-30de-11ec-88a5-fb21b48c9b4e","list_id":"simple_list","namespace_type":"single","type":"detection"}],"immutable":false,"type":"query","language":"kuery","query":"host.name: *","throttle":"no_actions","actions":[]}
{"_version":"WzQwMiwxXQ==","created_at":"2021-10-19T13:16:42.833Z","created_by":"elastic","description":"This is a sample endpoint type exception","id":"cd62f410-30de-11ec-88a5-fb21b48c9b4e","immutable":false,"list_id":"simple_list","name":"Sample Endpoint Exception List","namespace_type":"single","os_types":[],"tags":["user added string for a tag","malware"],"tie_breaker_id":"3e838fbe-9732-4014-9273-4b1b9bad5202","type":"detection","updated_at":"2021-10-19T13:16:42.835Z","updated_by":"elastic","version":1}
{"_version":"WzUwMiwxXQ==","comments":[],"created_at":"2021-10-19T13:17:22.974Z","created_by":"elastic","description":"This is a sample endpoint type exception","entries":[{"field":"actingProcess.file.signer","operator":"excluded","type":"exists"},{"field":"host.name","operator":"included","type":"match_any","value":["some host","another host"]}],"id":"e54ffbe0-30de-11ec-88a5-fb21b48c9b4e","item_id":"VjSeUlYdFEAjGNaQKq22CJwShH6wDbWUEYsHWJqnHIjrlSwSGsklcV5Ehwg6VgH055v47kkYadl93PvQOAhCAtjZEEU8zALxMOiS","list_id":"simple_list","name":"Sample Endpoint Exception List","namespace_type":"single","os_types":["linux"],"tags":["user added string for a tag","malware"],"tie_breaker_id":"b9e0070a-ef3b-402e-85fc-7b02497f1369","type":"simple","updated_at":"2021-10-19T13:17:22.976Z","updated_by":"elastic"}
{"exported_rules_count":1,"missing_rules":[],"missing_rules_count":0,"exported_exception_list_count":1,"exported_exception_list_item_count":1,"missing_exception_list_item_count":0,"missing_exception_list_items":[],"missing_exception_lists":[],"missing_exception_lists_count":0}
```

## PRs
- https://github.com/elastic/kibana/pull/115144

## Possible docs needing updates
- https://www.elastic.co/guide/en/security/current/rules-ui-management.html#import-export-rules-ui
- https://www.elastic.co/guide/en/security/current/rules-api-import.html
- https://www.elastic.co/guide/en/security/current/rules-api-export.html
- https://www.elastic.co/guide/en/security/current/detections-ui-exceptions.html#manage-exceptions

## Notes
- Exception imports are not yet supported, and are simply ignored on import at the moment. So exporting rues with exceptions and then reimporting that file, will not cause errors, but the exception lists themselves will not be considered on import
- Might want to consult with product as to what we want the wording to be here. Went ahead with exports in 7.16 so that users could then make use of it in migrating to 8.0 with exceptions import support then
- [Large value lists](https://www.elastic.co/guide/en/security/current/lists-api-overview.html), which can be referenced within exception list items, are not part of the export though import/export apis do exist for them separately

**Comment by joepeeples** [comment]
Per conversation with @yctercero, @jethr0null, @MikePaquette, @jmikell821: No documentation for 7.16, since import is not available yet. We'll wait until exceptions import via the UI is available (we're targeting 8.0), then backport into 7.16 docs the export functionality. Removing the `v7.16.0` label on this issue since it will not be part of 7.16 docs at release time, but keeping the issue open as something to act on for 8.0.

**Comment by joepeeples** [comment]
Importing exceptions with rules is now available as of 8.0.0, per https://github.com/elastic/kibana/pull/118816.

Also, https://github.com/elastic/kibana/pull/120837 adds new checkbox to import modal for overwriting existing exceptions.
